server/src/dev/resources/application-mirror.yml

```diff
@@ -0,0 +1,35 @@
+# --spring.profiles.include=ovsx,mirror
+ovsx:
+  data:
+    mirror:
+      enabled: true
+      server-url: https://open-vsx.org
+      requests-per-second: 5
+      user-name: mirror_user
+      schedule: '0/1 * * * *'
+      read-only:
+        disallowed-methods: POST, PUT, PATCH, DELETE
+        allowed-endpoints: /vscode/gallery/extensionquery, /admin/update-search-index, /api/-/query
+      excluded-extensions:
+        - vscode.*
+        - redhat.vscode-yaml
```

Comment:
Should we keep this? I know we can exclude `redhat.vscode-yaml` for our Gitpod mirror, but generally for folks self-hosting Open VSX, shouldn't we allow the YAML extension through as well? cc @jeanp413